platform/windows/os_windows.cpp

```diff
@@ -264,12 +264,19 @@ OS::Date OS_Windows::get_date(bool p_utc) const {
 		GetLocalTime(&systemtime);
 	}
 
+	//Get DST information from Windows.
+	TIME_ZONE_INFORMATION info;
+	bool daylight = false;
+	if (GetTimeZoneInformation(&info) == TIME_ZONE_ID_DAYLIGHT) {
+		daylight = true;
+	}
```

Comment:
Probably should not be done if `p_utc` is `true`.

Reply:
I believe that it it is impossible for this to incorrectly return it's daylight time in UTC? I may be incorrect though. [Documentation.](https://docs.microsoft.com/en-us/windows/win32/api/timezoneapi/nf-timezoneapi-gettimezoneinformation)

Reply:
That's if your time zone is UTC.

In this case, if `p_utc` parameter is `true`, it's gonna use UTC data from `GetSystemTime` and time zone info from your local timezone.

Reply:
Ohhhhh, my bad. Correcting this now.